### PR TITLE
Add playlist support

### LINKED ISSUE #10: Add playlist support

### PR DESCRIPTION
**Pull request by pimuzzo**
Fix https://github.com/pimuzzo/office-music/issues/10

### CODE REVIEW

File: play-server.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Start config - define your customizations inside .env.local
+source "$(dirname "$0")/.env"
+# End config
+
+rm -f songs/*
+
+while true; do
```
Comment (pimuzzo) [inline play-server.sh:9]: I would like to use another queue

Comment (pimuzzo) [reply play-server.sh:9]: I don't like server split in 2 different files

File: play-server.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  echo "Searching for songs to play..."
+  for file in songs/* ; do
+    if [[ $file != "songs/*" && ${file: -5} != ".part" ]]; then
+      filename=${file}
+      echo "Playing $filename..."
+      omxplayer -o "$SERVER_OMX_AUDIO_OUT_DEVICE" "$filename"
+      rm -f "$filename"
+    fi
+  done
+  sleep 1
```
Comment (pimuzzo) [inline play-server.sh]: 1s it is more explicit

Comment (pimuzzo) [inline play-server.sh]:
```suggestion
  sleep 1s
```

**Review by gionn — APPROVED** [review]

**Pull request merged**